modules/claims_api/lib/benefits_documents/configuration.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+require 'lighthouse/auth/client_credentials/service'
+
+module BenefitsDocuments
+  ##
+  #
+  class Configuration < Common::Client::Configuration::REST
```

Comment:
we talked about renaming these to better specify their usage from the claims api - are we still on board with that?

Reply:
Yes and I updated the existing files to be more embedded in the claims_api module but let me know if you prefer anything else, or more of the other service pulled into our module.